staging/src/k8s.io/apiextensions-apiserver/test/integration/table_test.go

```diff
@@ -97,6 +98,9 @@ func newTableInstance(name string) *unstructured.Unstructured {
 				"delta":   "hello",
 				"epsilon": []int64{1, 2, 3},
 				"zeta":    5,
+				"eta": []map[string]int{
```

Comment:
Is there any negative testing we can add here?  nil arrays, empty arrays, json errors we need to consider, maps of maps, etc?

Reply:
Added some more edge cases here - avoided maps of maps due to the relative complexity of writing tests with inconsistent output/ordering.